src/puppetlabs/trapperkeeper/services/authorization/authorization_service.clj

```diff
@@ -13,9 +13,9 @@
   [[:ConfigService get-in-config]]
   (init
     [this context]
-    (let [config (get-in-config [:authorization :rules])]
+    (let [config (get-in-config [:authorization])]
       (validate-auth-config! config)
-      (assoc-in context [:rules] (transform-config config))))
+      (assoc-in context [:rules] (transform-config (:rules config)))))
```

Comment:
I chose not to change what we persist in the service's context since we only need the version number during validation.  We'll need to change this when we add code that needs the version number after validation.